cabal.project

```diff
@@ -156,58 +268,14 @@ source-repository-package
     subdir: command-line
             core
 
--- Should follow cardano-wallet.
-source-repository-package
-    type: git
-    location: https://github.com/input-output-hk/cardano-base
-    tag: 0f3a867493059e650cda69e20a5cbf1ace289a57
-    subdir:
-            base-deriving-via
-            binary
-            binary/test
-            cardano-crypto-class
-            cardano-crypto-praos
-            cardano-crypto-tests
-            orphans-deriving-via
-            measures
-            strict-containers
-            slotting
-
--- Should follow cardano-wallet.
-source-repository-package
-    type: git
-    location: https://github.com/input-output-hk/cardano-crypto
-    tag: f73079303f663e028288f9f4a9e08bcca39a923e
-
--- Should follow cardano-node.
--- But in case there are failures with the plutus version, update to the latest
--- commit hash of the release/1.0.0 plutus branch.
-source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/plutus
-  tag: a56c96598b4b25c9e28215214d25189331087244
-  subdir:
-    plutus-core
-    plutus-ledger-api
-    plutus-tx
-    plutus-tx-plugin
-    prettyprinter-configurable
-    stubs/plutus-ghc-stub
-    word-array
-
--- Should follow cardano-node
-source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/ekg-forward
-  tag: 297cd9db5074339a2fb2e5ae7d0780debb670c63
-
 -- Should follow cardano-node
```

Comment:
If this one wasn't removed, that's because it was not released on CHaP?

Reply:
It was removed. The exact same version is present in CHaP https://input-output-hk.github.io/cardano-haskell-packages/package/ekg-forward-0.1.0/

Reply:
I'm actually referring to the line below: `cardano-config`

Reply:
Ah, yes, cardano-config is not in CHaP. I notice it's marked as deprecated and we don't actually seem to use it anywhere. I'll remove it.

Reply:
(it was once a dependecy of cardano-node but it has not been since January)